### DISCUSSION TITLE
Shelly RGBW2 white only .json

### DISCUSSION THREAD

**Discussion by mrmaximas** [RavenSystem/esp-homekit-devices#1664] [MEPLHAA Scripts]
I'm using some shellies rgbw2 in white-only mode via homebridge shelly plugin.
How to modify this working rgbw json {"c":{"l":2,"b":[{"g":13,"t":5}]},"a":[{"t":30,"r":15,"g":12,"v":14,"w":4,"b":[{"g":5,"p":0},{"g":5,"t":0},{"g":13}]}]} to white-only?
Please help )
Thanks!

**Comment by Hanimaniac** [comment]
Hello,
I use this json for the shelly rgbw2:
{"c":{"b":[{"g":0,"t":5}]},"a":[{"t":30,"g":[15],"b":[{"g":0}]},{"t":30,"g":[12],"b":[{"g":0}]},{"t":30,"g":[14],"b":[{"g":0}]},{"t":30,"g":[4],"b":[{"g":0}]}]}
There will be 4 dimmable lights in your Home App. 

Greets Martin

**Comment by mrmaximas** [comment]
Thanks. I will try to modify it for GPIO 4 only

**Comment by lukaszwitkowski** [comment]
Great job

**Comment by RavenSystem** [comment]
Single channel GPIO 4:

```json
{"c":{"l":2,"b":[{"g":13,"t":5}]},"a":[{"t":30,"g":[4],"b":[{"g":5,"p":0},{"g":5,"t":0},{"g":13}]}]}
```

**Reply by Hanimaniac** [reply]
With this json you totally underchallenge the shelly rgbw2 :)

**Comment by mrmaximas** [comment]
What is the right way to OTA flash HAA to Shelly RGBW2 with stock firmware?

**Comment by mrmaximas** [comment]
Thanks! It's works. Flashed to HAA via https://github.com/yaourdt/mgos-to-tasmota
